src/course4/homework/Challenge7.java

```diff
@@ -0,0 +1,28 @@
+package course4.homework;
+
+public class Challenge7 {
+
+    public static void main(String[] args) {
+
+        System.out.println("This code counts from 1 to 10 and back");
+
+        int firstNumber;
+
+        //starting the loop for counting from 1 to 10
+
+        for (firstNumber = 1; firstNumber <= 10; firstNumber++) //this sequence gets the starting number, checks if
+                                                                //our variable is lower or equal to 10 and, if
+                                                                //the condition checks, it increases its value by 1
+        {
```

Comment:
Java convention is to have `{` at the end of the declared statement, not the next line.
So you should have something like:
```
for (firstNumber = 1; firstNumber <= 10; firstNumber++) {
  ...
}
```
Also, `firstNumber` is not a appropriate one since it modifies in the `for` loop. Maybe rename it to just `number`.
However, nice to see you are using `for` loops even tough we haven't studied it yet. :)

Reply:
Fair point. I thought at firstNumber as being the first value of the number (in our case, 1) and that's why I gave it that name. For me, it made sense at that time :)). About the braces, I put them on the next line so that it I could follow them more easily and not miss any of them and after I was done with the code, I simply forgot to modify their position. I've modified the position of the braces and modified the variable's name.